### ISSUE TITLE
`ReorderableListView` crashes app in release mode in Windows

### ISSUE THREAD

**Issue by mjafartp** [flutter/flutter#109448]
Windows app closes when using [ReorderableListView](https://api.flutter.dev/flutter/material/ReorderableListView-class.html)

## Steps to Reproduce

Example code in https://api.flutter.dev/flutter/material/ReorderableListView-class.html also produces this error.

**Comment by exaby73** [comment]
Hi @mjafartp, please provide the output of `flutter doctor -v` as well as `flutter run -v`.

[1]: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by mjafartp** [comment]
flutter doctor -v
[√] Flutter (Channel stable, 3.0.5, on Microsoft Windows [Version 10.0.19044.1889], locale en-IN)
    • Flutter version 3.0.5 at C:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (4 weeks ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\xxx\AppData\Local\Android\sdk
    • Platform android-32, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.0)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.1.32210.238
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 4.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.70.1)
    • VS Code at C:\Users\xxx\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.36.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.1889]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 104.0.5112.81
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 104.0.1293.47

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!




flutter run --release -v
[ +112 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +665 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] f1875d570e39de09040c8f79aa13cc56baab8db1
[   +1 ms] executing: [C:\flutter/] git tag --points-at f1875d570e39de09040c8f79aa13cc56baab8db1
[ +111 ms] Exit code 0 from: git tag --points-at f1875d570e39de09040c8f79aa13cc56baab8db1
[   +1 ms] 3.0.5
[  +13 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +64 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [C:\flutter/] git ls-remote --get-url origin
[  +60 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +354 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[  +63 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +138 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +5 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +84 ms] executing: C:\Users\xxx\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +87 ms] List of devices attached
[  +10 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +7 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +136 ms] Skipping pub get: version match.
[ +164 ms] Generating C:\Apps\flutter_app_reorder\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +146 ms] Launching lib\main.dart on Windows in release mode...
[  +28 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[  +82 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +1 ms] [
             {
               "instanceId": "4e3ade35",
               "installDate": "2022-03-07T08:40:43Z",
               "installationName": "VisualStudio/17.1.0+32210.238",
               "installationPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community",
               "installationVersion": "17.1.32210.238",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2022",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.17.Release",
               "channelUri": "https://aka.ms/vs/17/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelUri": "https://aka.ms/vs/17/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.1#17.1.0",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=661288",
               "updateDate": "2022-03-07T08:40:43.8851258Z",
               "catalog": {
                 "buildBranch": "d17.1",
                 "buildVersion": "17.1.32210.238",
                 "id": "VisualStudio/17.1.0+32210.238",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "17.1.0",
                 "productLine": "Dev17",
                 "productLineVersion": "2022",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "0",
                 "productPreReleaseMilestoneSuffix": "7.0",
                 "productSemanticVersion": "17.1.0+32210.238",
                 "requiredEngineVersion": "3.1.2196.8931"
               },
               "properties": {
                 "campaignId": "2030",
                 "channelManifestId": "VisualStudio.17.Release/17.1.0+32210.238",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +5 ms] Building Windows application...
[  +18 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S C:\Apps\flutter_app_reorder\windows -B build\windows -G Visual Studio 17 2022
[  +76 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
[ +254 ms] -- Configuring done
[ +276 ms] -- Generating done
[  +17 ms] -- Build files have been written to: C:/Apps/flutter_app_reorder/build/windows
[  +34 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Release --target INSTALL --verbose
[ +208 ms] Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[  +97 ms] Build started 12-08-2022 05:26:23 PM.
[ +283 ms] Project "C:\Apps\flutter_app_reorder\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +1 ms] Project "C:\Apps\flutter_app_reorder\build\windows\INSTALL.vcxproj" (1) is building "C:\Apps\flutter_app_reorder\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[        ] PrepareForBuild:
[        ]   Creating directory "x64\Release\ZERO_CHECK\".
[   +4 ms]   Creating directory "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\".
[   +3 ms] InitializeBuildStatus:
[        ]   Creating "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +128 ms] CustomBuild:
[   +1 ms]   Checking Build System
[ +109 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[        ] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\ZERO_CHECK.vcxproj" (default targets).
[  +94 ms] Project "C:\Apps\flutter_app_reorder\build\windows\INSTALL.vcxproj" (1) is building "C:\Apps\flutter_app_reorder\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +1 ms] Project "C:\Apps\flutter_app_reorder\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Apps\flutter_app_reorder\build\windows\runner\flutter_app_reorder.vcxproj" (4) on node 1 (default targets).
[        ] Project "C:\Apps\flutter_app_reorder\build\windows\runner\flutter_app_reorder.vcxproj" (4) is building "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[        ] PrepareForBuild:
[        ]   Creating directory "x64\Release\flutter_assemble\".
[        ]   Creating directory "x64\Release\flutter_assemble\flutter_assemble.tlog\".
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Release\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[        ]   "The build of 'C:\Apps\flutter_app_reorder\build\windows\CMakeFiles\3d4530ca3fb988f8ba6d4f3af74794c6\flutter_assemble.rule' depends on 'C:\APPS\FLUTTER_APP_REORDER\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Apps\flutter_app_reorder\build\windows\CMakeFiles\0032049bb33318423123f669c6e72045\flutter_windows.dll.rule'. The items cannot be built in parallel."
[  +46 ms]   Generating C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_windows.dll, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_export.h, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_windows.h,
C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_messenger.h, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_plugin_registrar.h, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/flutter_texture_registrar.h,
C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Apps/flutter_app_reorder/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+1889 ms]   [ +105 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +646 ms]   [ +680 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms]   [        ] f1875d570e39de09040c8f79aa13cc56baab8db1
[        ]   [        ] executing: [C:\flutter/] git tag --points-at f1875d570e39de09040c8f79aa13cc56baab8db1
[ +124 ms]   [ +126 ms] Exit code 0 from: git tag --points-at f1875d570e39de09040c8f79aa13cc56baab8db1
[   +1 ms]   [        ] 3.0.5
[  +11 ms]   [  +12 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +86 ms]   [  +86 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[  +20 ms]   [        ] origin/stable
[        ]   [        ] executing: [C:\flutter/] git ls-remote --get-url origin
[  +74 ms]   [  +95 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms]   [        ] https://github.com/flutter/flutter.git
[  +82 ms]   [  +83 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[  +78 ms]   [  +78 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] stable
[ +104 ms]   [ +104 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms]   [   +5 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +145 ms]   [ +146 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ]   [   +4 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +52 ms]   [  +61 ms] Initializing file store
[  +36 ms]   [  +36 ms] Skipping target: gen_localizations
[  +20 ms]   [  +20 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: C:\Apps\flutter_app_reorder\.dart_tool\package_config_subset}
[   +5 ms]   [   +5 ms] unpack_windows: Starting due to {}
[+1441 ms]   [+1441 ms] unpack_windows: Complete
[+6679 ms]   [+6678 ms] gen_dart_plugin_registrant: Complete
[   +2 ms]   [   +3 ms] kernel_snapshot: Starting due to {}
[  +12 ms]   [  +12 ms] C:\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root C:\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk_product/ --target=flutter
--no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false -Ddart.vm.product=true --aot --tfa --packages C:\Apps\flutter_app_reorder\.dart_tool\package_config.json --output-dill
C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\app.dill --depfile C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\kernel_snapshot.d package:flutter_app_reorder/main.dart
[+18924 ms]   [+18924 ms] kernel_snapshot: Complete
[ +550 ms]   [ +550 ms] aot_elf_release: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
C:\flutter\packages\flutter_tools\lib\src\build_system\targets\common.dart,C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\app.dill,C:\flutter\bin\internal\engine.version,C:\flutter\bin\internal\engine.version,C:\flutter\bin\in
ternal\engine.version}
[   +8 ms]   [   +9 ms] executing: C:\flutter\bin\cache\artifacts\engine\windows-x64-release\gen_snapshot --deterministic --snapshot_kind=app-aot-elf --elf=C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\app.so --strip
C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\app.dill
[+8690 ms]   [+8690 ms] aot_elf_release: Complete
[  +31 ms]   [  +32 ms] windows_aot_bundle: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: C:\Apps\flutter_app_reorder\.dart_tool\flutter_build\0e0fe9a0fb49afabac5e57c7e2cc3524\app.so}
[   +8 ms]   [   +8 ms] windows_aot_bundle: Complete
[  +36 ms]   [  +36 ms] release_bundle_windows_assets: Starting due to {}
[ +151 ms]   [ +151 ms] release_bundle_windows_assets: Complete
[ +125 ms]   [ +125 ms] Persisting file store
[  +13 ms]   [  +13 ms] Done persisting file store
[  +17 ms]   [  +17 ms] build succeeded.
[  +13 ms]   [  +13 ms] "flutter assemble" took 37,010ms.
[ +173 ms]   [ +172 ms] ensureAnalyticsSent: 167ms
[   +2 ms]   [   +2 ms] Running shutdown hooks
[   +2 ms]   [        ] Shutdown hooks complete
[        ]   [   +1 ms] exiting with code 0
[  +82 ms]   Building Custom Rule C:/Apps/flutter_app_reorder/windows/flutter/CMakeLists.txt
[  +64 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Release\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Release\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[  +35 ms] Project "C:\Apps\flutter_app_reorder\build\windows\runner\flutter_app_reorder.vcxproj" (4) is building "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_wrapper_app.vcxproj" (6) on node 1 (default targets).
[   +1 ms] PrepareForBuild:
[        ]   Creating directory "flutter_wrapper_app.dir\Release\".
[   +1 ms]   Creating directory "C:\Apps\flutter_app_reorder\build\windows\flutter\Release\".
[        ]   Creating directory "flutter_wrapper_app.dir\Release\flutter_.E5D496A9.tlog\".
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Release\flutter_.E5D496A9.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +47 ms] CustomBuild:
[   +1 ms]   Building Custom Rule C:/Apps/flutter_app_reorder/windows/flutter/CMakeLists.txt
[ +116 ms] ClCompile:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\CL.exe /c /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\include /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral
/nologo /W4 /WX /diagnostics:column /O2 /Ob2 /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D NDEBUG /D _HAS_EXCEPTIONS=0 /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Release\"" /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17
/Fo"flutter_wrapper_app.dir\Release\\" /Fd"C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_app.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\core_implementations.cc C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\standard_codec.cc
C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\flutter_engine.cc C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\flutter_view_controller.cc
[ +348 ms]   core_implementations.cc
[+1039 ms]   standard_codec.cc
[+1728 ms]   flutter_engine.cc
[+1418 ms]   flutter_view_controller.cc
[+1363 ms]   Generating Code...
[+1468 ms] Lib:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\Lib.exe /OUT:"C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_app.lib" /NOLOGO /MACHINE:X64  /machine:x64
flutter_wrapper_app.dir\Release\core_implementations.obj
[   +1 ms]   flutter_wrapper_app.dir\Release\standard_codec.obj
[        ]   flutter_wrapper_app.dir\Release\flutter_engine.obj
[        ]   flutter_wrapper_app.dir\Release\flutter_view_controller.obj
[ +299 ms]   flutter_wrapper_app.vcxproj -> C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_app.lib
[  +16 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Release\flutter_.E5D496A9.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Release\flutter_.E5D496A9.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[   +4 ms] PrepareForBuild:
[        ]   Creating directory "flutter_app_reorder.dir\Release\".
[   +2 ms]   Creating directory "C:\Apps\flutter_app_reorder\build\windows\runner\Release\".
[        ]   Creating directory "flutter_app_reorder.dir\Release\flutter_.3C39DB61.tlog\".
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "flutter_app_reorder.dir\Release\flutter_.3C39DB61.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +44 ms] CustomBuild:
[   +1 ms]   Building Custom Rule C:/Apps/flutter_app_reorder/windows/runner/CMakeLists.txt
[  +52 ms] ClCompile:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\CL.exe /c /IC:\Apps\flutter_app_reorder\windows /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral
/IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\include /nologo /W4 /WX /diagnostics:column /O2 /Ob2 /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D NDEBUG /D _HAS_EXCEPTIONS=0 /D NOMINMAX /D UNICODE /D _UNICODE /D
"CMAKE_INTDIR=\"Release\"" /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"flutter_app_reorder.dir\Release\\" /Fd"flutter_app_reorder.dir\Release\vc143.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Apps\flutter_app_reorder\windows\runner\flutter_window.cpp C:\Apps\flutter_app_reorder\windows\runner\main.cpp C:\Apps\flutter_app_reorder\windows\runner\utils.cpp C:\Apps\flutter_app_reorder\windows\runner\win32_window.cpp
C:\Apps\flutter_app_reorder\windows\flutter\generated_plugin_registrant.cc
[  +55 ms]   flutter_window.cpp
[+1189 ms]   main.cpp
[+1139 ms]   utils.cpp
[+1269 ms]   win32_window.cpp
[+1015 ms]   generated_plugin_registrant.cc
[ +346 ms]   Generating Code...
[ +412 ms] ResourceCompile:
[   +1 ms]   C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\rc.exe /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D NDEBUG /D _HAS_EXCEPTIONS=0 /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\\\"Release\\\"" /l"0x0409"
/IC:\Apps\flutter_app_reorder\windows /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\include /nologo /fo"flutter_app_reorder.dir\Release\Runner.res"
C:\Apps\flutter_app_reorder\windows\runner\Runner.rc
[ +174 ms] Link:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Apps\flutter_app_reorder\build\windows\runner\Release\flutter_app_reorder.exe" /INCREMENTAL:NO /NOLOGO
..\flutter\Release\flutter_wrapper_app.lib C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST
/MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /manifestinput:C:\Apps\flutter_app_reorder\windows\runner\runner.exe.manifest /PDB:"C:/Apps/flutter_app_reorder/build/windows/runner/Release/flutter_app_reorder.pdb" /SUBSYSTEM:WINDOWS /TLBID:1
/DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/Apps/flutter_app_reorder/build/windows/runner/Release/flutter_app_reorder.lib" /MACHINE:X64  /machine:x64 flutter_app_reorder.dir\Release\Runner.res
[   +1 ms]   flutter_app_reorder.dir\Release\flutter_window.obj
[        ]   flutter_app_reorder.dir\Release\main.obj
[        ]   flutter_app_reorder.dir\Release\utils.obj
[        ]   flutter_app_reorder.dir\Release\win32_window.obj
[        ]   flutter_app_reorder.dir\Release\generated_plugin_registrant.obj
[ +518 ms]   flutter_app_reorder.vcxproj -> C:\Apps\flutter_app_reorder\build\windows\runner\Release\flutter_app_reorder.exe
[   +9 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_app_reorder.dir\Release\flutter_.3C39DB61.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_app_reorder.dir\Release\flutter_.3C39DB61.tlog\flutter_app_reorder.lastbuildstate".
[        ] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\runner\flutter_app_reorder.vcxproj" (default targets).
[  +34 ms] Project "C:\Apps\flutter_app_reorder\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (7) on node 1 (default targets).
[   +1 ms] PrepareForBuild:
[        ]   Creating directory "flutter_wrapper_plugin.dir\Release\".
[        ]   Creating directory "flutter_wrapper_plugin.dir\Release\flutter_.45D50B70.tlog\".
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Release\flutter_.45D50B70.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +73 ms] CustomBuild:
[   +1 ms]   Building Custom Rule C:/Apps/flutter_app_reorder/windows/flutter/CMakeLists.txt
[  +48 ms] ClCompile:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\CL.exe /c /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\include /IC:\Apps\flutter_app_reorder\windows\flutter\ephemeral
/nologo /W4 /WX /diagnostics:column /O2 /Ob2 /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D NDEBUG /D _HAS_EXCEPTIONS=0 /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Release\"" /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17
/Fo"flutter_wrapper_plugin.dir\Release\\" /Fd"C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_plugin.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\core_implementations.cc C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\standard_codec.cc
C:\Apps\flutter_app_reorder\windows\flutter\ephemeral\cpp_client_wrapper\plugin_registrar.cc
[  +53 ms]   core_implementations.cc
[ +816 ms]   standard_codec.cc
[+1461 ms]   plugin_registrar.cc
[ +837 ms]   Generating Code...
[+1404 ms] Lib:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\Lib.exe /OUT:"C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_plugin.lib" /NOLOGO /MACHINE:X64  /machine:x64
flutter_wrapper_plugin.dir\Release\core_implementations.obj
[        ]   flutter_wrapper_plugin.dir\Release\standard_codec.obj
[        ]   flutter_wrapper_plugin.dir\Release\plugin_registrar.obj
[ +496 ms]   flutter_wrapper_plugin.vcxproj -> C:\Apps\flutter_app_reorder\build\windows\flutter\Release\flutter_wrapper_plugin.lib
[  +14 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Release\flutter_.45D50B70.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_plugin.dir\Release\flutter_.45D50B70.tlog\flutter_wrapper_plugin.lastbuildstate".
[   +2 ms] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +14 ms] PrepareForBuild:
[        ]   Creating directory "x64\Release\ALL_BUILD\".
[   +1 ms]   Creating directory "x64\Release\ALL_BUILD\ALL_BUILD.tlog\".
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "x64\Release\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +80 ms] CustomBuild:
[   +1 ms]   Building Custom Rule C:/Apps/flutter_app_reorder/windows/CMakeLists.txt
[  +86 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Release\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Release\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +1 ms] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +2 ms] PrepareForBuild:
[        ]   Creating directory "x64\Release\INSTALL\".
[   +2 ms]   Creating directory "x64\Release\INSTALL\INSTALL.tlog\".
[  +36 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Release\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +126 ms] PostBuildEvent:
[   +1 ms]   setlocal
[        ]   "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Release -P cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +61 ms]   -- Install configuration: "Release"
[  +76 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/icudtl.dat
[   +6 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/flutter_windows.dll
[  +63 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets
[   +3 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/AssetManifest.json
[   +2 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/FontManifest.json
[   +2 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/fonts
[   +1 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +8 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/NOTICES.Z
[  +44 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/packages
[   +1 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/packages/cupertino_icons
[   +2 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/packages/cupertino_icons/assets
[   +1 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[ +202 ms]   -- Installing: C:/Apps/flutter_app_reorder/build/windows/runner/Release/data/app.so
[  +73 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Release\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Release\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[        ] Done Building Project "C:\Apps\flutter_app_reorder\build\windows\INSTALL.vcxproj" (default targets).
[        ] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:01:01.47
[  +40 ms] Building Windows application... (completed in 62.5s)
[  +34 ms] Application running.
[   +5 ms] Flutter run key commands.
[   +2 ms] h List all available interactive commands.
[   +3 ms] c Clear the screen
[   +3 ms] q Quit (terminate the application on the device).

**Comment by mjafartp** [comment]
in Debug following error message we getting. 
![Error Screenshot](https://user-images.githubusercontent.com/5754379/184352527-5a537b52-145a-422b-9e56-9cde2c0d9581.png)

**Comment by exaby73** [comment]
I am unable to reproduce this issue. I have found a similar issue:

- https://github.com/flutter/flutter/issues/103808

Does this one describe your issue?

**Comment by mjafartp** [comment]
@exaby73 Yes this was the issue.

**Comment by exaby73** [comment]
@mjafartp Which one? The currently open issue (https://github.com/flutter/flutter/issues/103808)

**Comment by exaby73** [comment]
Marking as duplicate of https://github.com/flutter/flutter/issues/103808
